1.8/administration/tls-ssl/index.md

```diff
@@ -0,0 +1,8 @@
+---
+post_title: Securing communication with TLS
+menu_order: 7
+---
+
+You can configure trusted communications with your DC/OS cluster by [setting up a proxy](/1.9/administration/haproxy-adminrouter/) between the Admin Router and user agent requests coming in from outside of the cluster. 
```

Comment:
"set up secure HTTPS communication using a custom server certificate" instead of "configure trusted communications"